web-frontend/modules/automation/store/automationWorkflowNode.js

```diff
@@ -361,6 +364,19 @@ const actions = {
 
     commit('DELETE_ITEM', { workflow, nodeId })
 
+    const automation = rootGetters['application/get'](workflow.automation_id)
+    if (automation) {
+      // Refresh updated workflow (simulate_until_node_id could have changed)
+      await dispatch(
```

Comment:
Could we avoid the full refresh here and just put Null in the `simulate_until_node` here?